Binary files libsquish/.DS_Store and libsquish-hap/.DS_Store differ
diff -rupN libsquish/GNUmakefile libsquish-hap/GNUmakefile
--- libsquish/GNUmakefile	2012-03-26 08:23:32.000000000 +0100
+++ libsquish-hap/GNUmakefile	2012-09-29 03:40:33.000000000 +0100
@@ -15,8 +15,8 @@ ifeq ($(HOSTTYPE),intel-pc) # intel-pc =
 endif
 
 ifeq ($(USE_SHARED),1)
-   SOLIB = libsquish.so.$(SOVER)
-   LIB = $(SOLIB).0
+   SOLIB = libsquish.$(SOVER).dylib
+   LIB = $(SOLIB)
    CPPFLAGS += -fPIC
 else
    LIB = libsquish.a
@@ -25,16 +25,16 @@ endif
 all: $(LIB)
 
 install: $(LIB)
-	install squish.h $(INSTALL_DIR)/include
-	install $(LIB) $(INSTALL_DIR)/lib
+	install squish.h $(SQUISH_INSTALL_DIR)/include
+	install $(LIB) $(SQUISH_INSTALL_DIR)/lib
 
 uninstall:
-	$(RM) $(INSTALL_DIR)/include/squish.h
-	$(RM) $(INSTALL_DIR)/lib/$(LIB)
+	$(RM) $(SQUISH_INSTALL_DIR)/include/squish.h
+	$(RM) $(SQUISH_INSTALL_DIR)/lib/$(LIB)
 
 $(LIB): $(OBJ)
 ifeq ($(USE_SHARED),1)
-	$(CXX) -shared -Wl,-soname,$(SOLIB) -o $@ $(OBJ)
+	$(CXX) -dynamiclib -Wl,-install_name,@rpath/$(SOLIB) -o $@ $(OBJ)
 else
 	$(AR) cr $@ $?
 	ranlib $@
diff -rupN libsquish/config libsquish-hap/config
--- libsquish/config	2012-03-24 11:35:01.000000000 +0000
+++ libsquish-hap/config	2012-09-30 00:16:20.000000000 +0100
@@ -21,4 +21,4 @@ ifeq ($(USE_SSE),1)
 endif
 
 # where should we install to
-INSTALL_DIR ?= /usr/local
+SQUISH_INSTALL_DIR ?= $(DERIVED_FILE_DIR)/squish-install
